civictechprojects/static/css/partials/_AboutProject.scss

```diff
@@ -177,6 +177,9 @@
     }
 
   }
+  .AboutProjects-hide {
+    display: none;
```

Comment:
Using CSS to hide elements works, but the convention we've adopted for the project is to just not render hidden elements in the first place.